frame/democracy/src/lib.rs

```diff
@@ -1716,10 +1630,9 @@ impl<T: Trait> Module<T> {
 	///   `ReferendumCount`, `LowestUnbaked`
 	/// - Db writes: `PublicProps`, `account`, `ReferendumCount`, `DepositOf`, `ReferendumInfoOf`
 	/// - Db reads per R: `DepositOf`, `ReferendumInfoOf`
-	/// - Base Weight: 58.58 + 10.9 * R µs
 	/// # </weight>
 	fn begin_block(now: T::BlockNumber) -> Result<Weight, DispatchError> {
-		let mut weight = 60_000_000 + T::DbWeight::get().reads_writes(6, 5);
+		let mut weight = 0;
```

Comment:
What happened to this weight?

Reply:
previously it was split between here and below:
```rust
	let r = Weight::from(last.saturating_sub(next));
		weight += 11_000_000 * r + T::DbWeight::get().reads(2 * r);
```
but now everything is moved below, actually we can move it here, but then we need to fetch next and last two times

Reply:
Okay, I dont mind if it moves, just that it doesn't disappear.